repo/content/caching_options.go

```diff
@@ -29,6 +29,7 @@ type CachingOptions struct {
 	MinContentSweepAge          DurationSeconds `json:"minContentSweepAge,omitempty"`
 	MinIndexSweepAge            DurationSeconds `json:"minIndexSweepAge,omitempty"`
 	HMACSecret                  []byte          `json:"-"`
+	KeyDerivationAlgorithm      string          `json:"keyDerivationAlgorithm,omitempty"`
```

Comment:
This option is confusing. It conveys that the key derivation algorithm would apply when deriving the cache encryption key for all the different caches (use cases). However this is used **only** in a very specific scenario, that is for client caches when connecting to a kopia repository server (that is, via the repository server API).

Reply:
@bathina2 What would be a good name for this field?

Reply:
I'm also wondering if this struct is the right place for this field. Thoughts?